test/functional/terminal/job_spec.lua

```diff
@@ -31,9 +30,6 @@ describe('job handler', function()
         elseif 'stderr' ==# a:event
           put =join(a:data,"\n")
         elseif 'exit' ==# a:event
-          " Length of received data.
-          call setline('.', 'bytes: '.len(join(g:job_stdout[0:-2],"\n")))
```

Comment:
Why would there be inconsistencies in the output of a non-interactive shell that just cats a file?